examples/test_steps_blending_rmi.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+
+from matplotlib import cm, pyplot as plt
+import numpy as np
+import xarray as xr
+import os
+from datetime import datetime
+from pprint import pprint
+from pysteps import io, rcparams
+from pysteps.utils import conversion, transformation, reprojection
+from pysteps.visualization import plot_precip_field
+
+# Set workdir and number of output time steps
+work_dir = "c:\\Users\\imhof_rn\\pysteps"
+os.chdir(work_dir)
+n_timesteps = 36
+
+# Give date as input
+date_str = "202107141010"  # input("Give date and time (e.g.: 201609281600):    ")
+
+date = datetime.strptime(date_str, "%Y%m%d%H%M")
+data_source = rcparams.data_sources["rmi"]
+
+# Import the radar composite to reproject NWP
+root_path = data_source["root_path"]
+path_fmt = data_source["path_fmt"]
+fn_pattern = data_source["fn_pattern"]
+fn_ext = data_source["fn_ext"]
+timestep = data_source["timestep"]
+importer_name = data_source["importer"]
+importer_kwargs = data_source["importer_kwargs"]
+
+# Find the radar files in the archive
+fns = io.find_by_date(
+    date, root_path, path_fmt, fn_pattern, fn_ext, timestep, num_prev_files=2
+)
+
+importer = io.get_method(importer_name, "importer")
+R_input = io.read_timeseries(fns, importer, legacy=False, **importer_kwargs)
```

Comment:
to import test data, you should use the helper function [get_precipitation_fields](https://github.com/pySTEPS/pysteps/blob/229dd9394b22423ec60741edc6f2ca10fcac274b/pysteps/tests/helpers.py#L25)
example [here](https://github.com/pySTEPS/pysteps/blob/229dd9394b22423ec60741edc6f2ca10fcac274b/pysteps/tests/test_noise_fftgenerators.py#L7).